website/blog/2024-08-06-langgraph-pyfunc/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: LangGraph with Custom PyFunc 
+tags: [genai, mlops]
+slug: mlflow-tracing
+authors: [mlflow-maintainers]
+thumbnail: img/blog/release-candidates.png
+---
+
+In this blog, we'll guide you through creating a LangGraph chatbot within an MLflow custom PyFunc.
+
+### What is a Custom PyFunc?
+
+While MLflow strives to cover the many popular machine learning libraries, there are libraries and pieces of functionality within supported libraries that are not natively supported. For these cases, when users want MLflow functionality they can create a custom PyFunc model. 
```

Comment:
Can we adjust the tone to not mention "not supported" and instead refer to the proliferation of packages that support Agent Frameworks and how even if there isn't a popular package YET supported natively, you can still get all of the benefits of MLflow tracking and deployment by using a custom pyfunc?